lib/config/defaults.js

```diff
@@ -595,9 +631,14 @@ const applyResolveDefaults = (resolve, { cache, mjs, webTarget }) => {
 	});
 	F(resolve, "mainFiles", () => ["index"]);
 	F(resolve, "aliasFields", () => (webTarget ? ["browser"] : []));
+	F(resolve, "exportsFields", () => ["exports"]);
 	F(resolve, "mainFields", () =>
 		webTarget ? ["browser", "module", "main"] : ["module", "main"]
 	);
+	F(resolve.byDependency, "commonjs", () => ({}));
+	F(resolve.byDependency.commonjs, "conditionNames", () => ["require", "..."]);
+	F(resolve.byDependency, "esm", () => ({}));
+	F(resolve.byDependency.esm, "conditionNames", () => ["import", "..."]);
```

Comment:
So we will have `import`/`webpack`/`browser`?

Reply:
yes for `webTarget` and `import`. I am afraid that you mean resolution order. Resolution order follows keys order in `package.json`, e.g. for
```
{
    "webpack": "...",
    "import": "..."
}
```
webpack target will be used in this case